lib/ControlledVocab/ControlledVocab.js

```diff
@@ -318,8 +319,8 @@ class ControlledVocab extends React.Component {
     }
   }
 
-  handlePaneFocus({ paneRef }) {
-    getFirstFocusable(paneRef.current)?.focus();
+  handlePaneFocus({ paneTitleRef }) {
+    paneTitleRef.current.focus();
```

Comment:
![image](https://user-images.githubusercontent.com/86330150/153608180-430f0c0f-d18c-4fe1-93ad-adf8017afa90.png)
I've added If condition because of eslint settings